src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -76,7 +76,7 @@ object AstTools {
 
       astNode match {
         case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlStringType => WdlString(t.getSourceString)
-        case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString)
+        case t: Terminal if t.getTerminalStr == "string" && wdlType == WdlFileType => WdlFile(t.getSourceString, false)
```

Comment:
I think at the least I'll make the true/false into an enum. I also agree about hiding it from the constructor unless totally necessary. Subtyping the WDL type turned out to be non-obvious so I'll see if I can get it working first like this then revisit that decision